src/ServicePulse.Host/vue/src/components/monitoring/EndpointList.vue

```diff
@@ -30,23 +35,58 @@ onMounted(async () => {
     <!--Table headings-->
     <div class="table-head-row">
       <div class="table-first-col">
-        <endpoint-list-sortable-column>Endpoint name</endpoint-list-sortable-column>
+        <EndpointListSortableColumn :is-active="activeColumn === 'name'" :sort-by="'name'" @click="updateActiveColumn('name')">Endpoint name</EndpointListSortableColumn>
       </div>
       <div class="table-col">
-        <endpoint-list-sortable-column v-tooltip title="Queue length: The number of messages waiting to be processed in the input queue(s) of the endpoint.">Queue Length<template #unit>(MSGS)</template></endpoint-list-sortable-column>
+        <EndpointListSortableColumn
+          :is-active="activeColumn === 'queueLength'"
+          :sort-by="'queueLength'"
+          @click="updateActiveColumn('queueLength')"
+          v-tooltip
+          title="Queue length: The number of messages waiting to be processed in the input queue(s) of the endpoint."
+          >Queue Length<template #unit>(MSGS)</template></EndpointListSortableColumn
```

Comment:
I'm having a little trouble with prettier.  the sortableColumn "Critical Time" is the only column that put each parameter on a new line.  When I tried to apply that to the other columns when I saved the file it would automatically set everything back.  Maybe I don't have it configured correctly.

Reply:
no, prettier will single line below `printWidth` which is currently set to 250 in order to reduce the conflicts on the monitoring branch. Now that the componentisation release is complete, we can set this down to something sane, like 120